frontend/src/v4/routes/components/textField/textField.styles.ts

```diff
@@ -77,6 +77,7 @@ export const StyledMarkdownField = styled(MarkdownField)`
 		font-size: 14px;
 		overflow: hidden;
 
+		${({ $isPlaceholder }) => $isPlaceholder && css`color: #C1C8D5;`} // TODO: fix after new palette is released
```

Comment:
It would be good to use the `isV5()` here so that you do not affect the v4 styling. Alternatively, use the `isV5()` when you pass the $isPlaceholder prop so that it is always false for v4
> In `textField.component.tsx`
>      $isPlaceholder={!this.fieldValue && isV5()}